desktop/apps/consignments2/components/step_marker/index.styl

```diff
@@ -1,32 +1,61 @@
 .consignments2-step-marker
   margin-top 80px
+  margin-bottom 100px
+  margin-left 50px
+  margin-right 50px
 
   &_mobile
     margin-top 60px
 
   &__steps
-    margin 0 auto
-    max-width 800px
+    ul
+      max-width 750px
+      margin 0 auto
+      align-content center
+      align-items center
+      display flex
+      justify-content space-around
+
+  &__step
+    content ' '
     display flex
-    justify-content space-between
+    flex-grow 1
+    margin 0
+    position relative
+    text-align right
+    z-index -1
+    border 1px dashed purple-color
 
-  &__dot
-    margin 0 auto
-    background gray-lighter-color
-    width 10px
-    height 10px
-    border-radius 50%
-    margin-bottom 8px
+    &::before
+      background purple-color
+      border-radius 50%
+      height 12px
+      position absolute
+      text-align center
+      top -6px
+      width 12px
+      left -2px
+      content ' '
 
-  &__label
-    avant-garde-size('s-headline')
-    color gray-lighter-color
+    &:last-child
+      flex-basis 0
+      flex-grow 0
+      flex-shrink 1
 
-  &__step
-    max-width 140px
-    text-align center
     &_active
-      .consignments2-step-marker__label
-        color purple-color
-      .consignments2-step-marker__dot
-        background purple-color
+      border 1px dashed gray-lighter-color
+
+    &_active ~ li
```

Comment:
It's a tricky little sibling selector (they describe it better): https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_selectors